tools/ka-customizer-runner

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash -ex
 
+BRANCH=$1
```

Comment:
Nope, not intended for human invocation. ka-customizer runs this, and it handles chaining commands easier done in bash than gjs.

Presumably if you are a human and comfortable with flatpak and the command line, you can just use the flatpak and knowledge app cli options directly.